spec/telephone_appointments/response_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe TelephoneAppointments::Response do
+  subject { described_class.new(response) }
+
+  context 'when the response is valid' do
+    let(:response) { Net::HTTPSuccess.new(nil, nil, nil) }
+
+    it 'should be a success' do
+      expect(subject).to be_success
+    end
+
+    it 'does not have any errors' do
+      expect(subject.errors).to eq({})
+    end
+  end
+
+  context 'when the response is not valid' do
+    let(:response) { double(body: { trade: 'invalid' }.to_json) }
+
+    it 'should be a success' do
+      expect(subject).not_to be_success
+    end
+
+    it 'does not have any errors' do
```

Comment:
I think you've copy pasted this spec description as it doesn't match with what's actually happening here.